tests/test_client/tests.py

```diff
@@ -85,6 +87,83 @@ def test_post(self):
         self.assertEqual(response.templates[0].name, 'POST Template')
         self.assertContains(response, 'Data received')
 
+    def test_json_post(self):
+        """ POST some JSON data to a view """
+        post_data = {
+            'value': 37
+        }
+        response = self.client.post('/json_view/', post_data, content_type=JSON_CONTENT)
+
+        # Check some response details
```

Comment:
I don't see much value in this comment. IMO it can be removed from all tests.

Reply:
Copy paste, will remove.